include/rocksdb/options.h

```diff
@@ -1600,6 +1600,21 @@ struct ReadOptions {
   // Default: false
   bool adaptive_readahead;
 
+  // For file reads associated with this operation, charge the internal rate
+  // limiter (see `DBOptions::rate_limiter`) at the specified priority. The
+  // special value `Env::IO_TOTAL` disables charging the rate limiter.
+  //
+  // The rate limiting is bypassed no matter this option's value for
+  // plain/cuckoo table reads, old BlobDB reads, and generally small reads like
+  // for file headers/footers.
+  //
+  // APIs that rely on `RandomAccessFile::MultiRead()`, such as
+  // `DB::MultiGet()`, return `Status::NotSupported` when
+  // `priority != Env::IO_TOTAL`.
+  //
+  // Default: `Env::IO_TOTAL`.
+  Env::IOPriority priority = Env::IO_TOTAL;
```

Comment:
Is there a reason why we go for a general name `priority`, instead of something like `rate_limiter_priority` which might better avoid future collision with priority related field being added here?

Reply:
Done.